### DISCUSSION TITLE
Go to definition when you're working on CPython itself

### DISCUSSION THREAD

**Discussion by gvanrossum** [microsoft/pylance-release#2588] [Enhancement]
I sometimes use VS Code to debug parts of the CPython stdlib (the parts written in Python). This works great until I hit F12 to go to the definition of a stdlib function in a different module, since it shows me the module in the *installed* Python version rather than the corresponding module in the CPython repo that I am working on. (In particular, the entire stdlib lives under "Lib/" in the repo.)

Is there a setting that would modify Go to definition this way? Or could we add one?

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Comment by heejaechang** [comment]
if you set extraPath to the `Lib` folder, it might just work?
